src/js/tabs/cli.js

```diff
@@ -107,6 +107,16 @@ TABS.cli.initialize = function (callback) {
             $('.tab-cli .window .wrapper').empty();
         });
 
+        $('.tab-cli .copy').click(function() {
+            try {
+                let gui = require('nw.gui'),
+                    clipboard = gui.Clipboard.get();
+                clipboard.set(self.outputHistory, "text");
+            } catch (ex) {
```

Comment:
This will not work and throw every time when used in the Chrome web app. What we should do is check the availability of the NW GUI on tab load, and gray out / hide the button if NW is not available.